src/commcare_cloud/commands/deploy.py

```diff
@@ -28,14 +33,39 @@ class Deploy(CommandBase):
     def run(self, args, unknown_args):
         check_branch(args)
         environment = get_environment(args.env_name)
-        commcare_branch = self._confirm_commcare_branch(environment, args.commcare_branch)
-        fab_func_args = self.get_fab_func_args(args)
+        commcare_branch = self._confirm_commcare_branch(environment, args.commcare_branch,
+                                                        quiet=args.quiet)
+        if args.component == 'commcare':
+            print(blue("You are about to deploy commcare"))
+            print(blue("branch: {}".format(commcare_branch)))
+            if ask('Deploy commcare?', quiet=args.quiet):
+                print(yellow("Formplayer will not be deployed right now,"))
+                print(yellow("but we recommend deploying formplayer about once a month as well."))
+                print(yellow("It causes about 1 minute of service interruption to Web Apps and App Preview,"))
+                print(yellow("but keeps these services up to date."))
+                print(yellow("You can do so by running `commcare-cloud <env> deploy formplayer`"))
```

Comment:
how long do you plan to keep this warning (info?) for?

Also do we have a standard of what colors in the terminal means? It might e good to have a page on the meaning behind each one so that we stay consistent

Reply:
Having quick color guide (that includes something like 3-5 colors) would be useful for consistency. So far my personal color guide has been something like
- never use explicit `black` or `white` (since you don't know the user's background color)
- use `cyan` for printing out which other shell commands are being run in the background (like the underlying `ansible`, `ansible-playbook`, `fab`, `terraform`, etc. bash command being run)
- default/no text color for "Do you want to X [y/N]?" prompts
Things we've been less consistent about are when to use `yellow`, `blue`, `magenta`, and `red`. I might propose the following guidelines for them
- `yellow` for PSAs/notices, and for highlighting the part of a longer message that is most likely to be useful as a tl;dr or refresher.
- `blue` for summary statements that are part of a longer scrolling output (like "Your private release is located here:" during deploy)
- `magenta` for warnings likely to be problematic
- `red` for errors

Thoughts?

Reply:
On second thought, I think that this shouldn't need docs. What if we had a `messages.py` where `warning = yellow` or whatever and we didn't use colors, but instead used `warning(text)`, `notice(text)` and so on. That would likely be more maintanable and more obvious to others without needing to go somewhere else for docs.

Also for conformity with other tools, we probably want to use ansible's color selections: https://docs.ansible.com/ansible/latest/reference_appendices/config.html?highlight=colors#color-changed. For example `purple` / `magenta` is used for deprecation warnings. (alternatively we could specify whatever we want in `ansible.cfg`)

Reply:
OK followed up here: https://github.com/dimagi/commcare-cloud/pull/3245.

Reply:
As for
> Also for conformity with other tools, we probably want to use ansible's color selections: https://docs.ansible.com/ansible/latest/reference_appendices/config.html?highlight=colors#color-changed. For example purple / magenta is used for deprecation warnings. (alternatively we could specify whatever we want in ansible.cfg)

I think that's a good idea, and I think just adding the semantic aliases for the colors we already use is a good start towards that. In that PR I also copied in the ansible color scheme info as a comment for consideration when choosing colors. Since it wasn't super clear cut how our categories relate to theirs in every case, I didn't try to reconcile them in the same PR, but that could be a follow up